huaweicloud/data_source_huaweicloud_networking_secgroup.go

```diff
@@ -166,10 +166,8 @@ func dataSourceNetworkingSecGroupReadV1(_ context.Context, d *schema.ResourceDat
 	resp := filterGroups[0].(v1groups.SecurityGroup)
 	d.SetId(resp.ID)
 
-	rules, err := flattenSecurityGroupRulesV1(&resp)
-	if err != nil {
-		return diag.FromErr(err)
-	}
+	rules := flattenSecurityGroupRulesV1(&resp)
+
```

Comment:
Lines 169 and 171 are logically tight, suggest to remove line 170.

Reply:
ok